poly/deal.go

```diff
@@ -234,10 +234,10 @@ type Promise struct {
  *    secretPair.Suite == longPair.Suite
  *
  * Returns
- *   A newly constructed Promise
+ *   A newly constructed Deal
  */
-func (p *Promise) ConstructPromise(secretPair *config.KeyPair,
-	longPair *config.KeyPair, t, r int, insurers []abstract.Point) *Promise {
+func (p *Deal) ConstructDeal(secretPair *config.KeyPair,
```

Comment:
I think we should talk about this more in depth when we start refactoring the promise / deal code ...?  Like we said yesterday, there are others issues in this code and i think It deserve its proper pull request.